lib/components/base/Pagination.vue

```diff
@@ -0,0 +1,202 @@
+<template>
```

Comment:
![image](https://user-images.githubusercontent.com/6166773/221405212-770c8988-ed23-4487-801b-ff53f398443f.png)

It'd be kinda nice if they were perfect circles when it was only a 1-digit number, right now they vary in width a bit depending on how wide the digit is.